protoc_plugin/lib/indenting_writer.dart

```diff
@@ -6,6 +6,8 @@ library protoc.indenting_writer;
 
 import 'src/descriptor.pb.dart';
 
+typedef _Body = void Function();
```

Comment:
Would it be better to just inline this instead of creating the private typedef, especially since the signature is so simple? I'm thinking of the avoid_private_typedef_functions lint and this preference: https://critique-ng.corp.google.com/cl/326560963

Reply:
You're right